### PR TITLE
refactor: unify and group together the creation of the bouncers and assemblers

### PR DESCRIPTION
**Pull request by rumenov**

### CODE REVIEW

File: rs/consensus/src/certification/certifier.rs [hunk 1]
```diff
@@ -49,18 +56,20 @@ pub struct CertifierImpl {
 
 /// The Certification component, processing the changes on the certification
 /// pool and submitting the corresponding change sets.
```
Comment (kpop-dfinity) [inline rs/consensus/src/certification/certifier.rs:58]: nit: the comment needs to be updated

File: rs/consensus/src/consensus.rs [hunk 2]
```diff
@@ -574,18 +576,18 @@ fn add_to_validated<T: ConsensusMessageHashable>(timestamp: Time, msg: Option<T>
 }
 
 /// Implement Consensus Gossip interface.
```
Comment (kpop-dfinity) [inline rs/consensus/src/consensus.rs:578]: ditto

File: rs/consensus/src/consensus.rs [hunk 3]
```diff
@@ -148,11 +148,13 @@ impl ConsensusImpl {
         message_routing: Arc<dyn MessageRouting>,
         state_manager: Arc<dyn StateManager<State = ReplicatedState>>,
         time_source: Arc<dyn TimeSource>,
-        stable_registry_version_age: Duration,
+        registry_poll_delay_duration_ms: u64,
```
Comment (kpop-dfinity) [inline rs/consensus/src/consensus.rs:151]: nit: could we change the type to `Duration`?

File: rs/consensus/src/idkg.rs [hunk 4]
```diff
@@ -432,16 +432,16 @@ impl<T: IDkgPool> ChangeSetProducer<T> for IDkgImpl {
     }
 }
 
-/// `IDkgGossipImpl` implements the bouncer function and other gossip related
+/// `IDkgBouncer` implements the bouncer function and other gossip related
 /// functionality
```
Comment (kpop-dfinity) [inline rs/consensus/src/idkg.rs:436]: nit:
```suggestion
/// `IDkgBouncer` implements the bouncer function
```

**Review by kpop-dfinity — APPROVED** [review]
nice

File: rs/consensus/src/consensus.rs (hunk 3 above)
Comment (rumenov) [reply rs/consensus/src/consensus.rs:151]: Let me do this in a follow-up MR because passing this constant from so deep also doesn't make much sense.

**Review by tthebst — APPROVED** [review]

**Pull request merged**